.github/workflows/build.yml

```diff
@@ -4,15 +4,19 @@ on:
   push:
     branches:
       - main
+      - 'release/**'
   pull_request:
     branches:
       - main
+      - 'release/**'
 
 jobs:
-  test-and-build:
+  test-and-build-linux:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
We had a problem getting the tags (`git describe --tags`) on our fork because the default value of `fetch-depth` is 1, but when setting it to 0, everything works as expected.
For more details, please see https://buildpacks.slack.com/archives/CSFE49VGR/p1600804742079700.

Reply:
For sure! We run `git describe --tags` to assign a version when on the `main` branch, so we need the repo to be checked out with history. From [actions/checkout](https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches): 

> Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set `fetch-depth: 0` to fetch all history for all branches and tags.